bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ChargedHiggs_taunu/ChargedHiggs_taunu_heavy_NLO/ChargedHiggs_taunu_heavy_NLO_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+import model 2HDMtII_NLO
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```

Comment:
Can you please remove these `define` lines? You are not using these definitions to generate the process.